app/grandchallenge/annotations/models.py

```diff
@@ -0,0 +1,167 @@
+from django.db import models
+from django.contrib.postgres.fields import ArrayField
+from django.utils import timezone
+from grandchallenge.core.models import UUIDModel
+from grandchallenge.cases.models import Image
+from django.contrib.auth import get_user_model
+
+
+class AbstractAnnotationModel(UUIDModel):
+    """
+    Abstract model for an annotation linking to a grader.
+    Overrides the created attribute from UUIDModel to allow the value to be set to a specific value on save.
+    See: https://docs.djangoproject.com/en/2.1/ref/models/fields/#django.db.models.DateField.auto_now_add
+    """
+
+    grader = models.ForeignKey(get_user_model(), on_delete=models.CASCADE)
+    # Override inherited 'created' attribute
+    created = models.DateTimeField(default=timezone.now)
+
+    class Meta(UUIDModel.Meta):
+        abstract = True
+
+
+class AbstractImageAnnotationModel(AbstractAnnotationModel):
+    """
+    Abstract model for annotation linking to a single image
+    """
+
+    image = models.ForeignKey(Image, on_delete=models.CASCADE)
+
+    def __str__(self):
+        return "<{} by {} on {} for {}>".format(
+            self.__class__.__name__,
+            self.grader.username,
+            self.created.strftime("%Y-%m-%d at %H:%M:%S"),
+            self.image,
+        )
+
+    class Meta(UUIDModel.Meta):
```

Comment:
It is not necessary right now because `UUIDModel.Meta` only contains the `abstract = True` statement that is not inherited. However, if another line is added, for instance ordering on modified field, that will only be inherited if the Meta is subclassed. I think that is useful behaviour

Reply:
Removed this subclassing (e4eeabb)